src/main/minio.js

```diff
@@ -941,14 +945,14 @@ export class Client {
       headers.range = range
     }
 
-    var expectedStatus = 200
+    var expectedStatusCodes = [200]
     if (range) {
-      expectedStatus = 206
+      expectedStatusCodes.push(206)
```

Comment:
if range is set, the status code cannot be 200 right

Reply:
It is set such that 200 is allowed. HTTP spec doesn't seem to explicit deny it.